frontend/packages/console-app/src/plugin.tsx

```diff
@@ -114,7 +114,7 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.default),
       // t('console-app~Control Plane status')
       popupTitle: '%console-app~Control Plane status%',
-      disallowedProviders: ['IBMCloud'],
+      disallowedProviders: [{ provider: 'IBMCloud', controlPlaneTopology: 'External' }],
```

Comment:
all providers should be disallowed if controlPlaneTopology == 'External'

Reply:
![Screen Shot 2022-04-26 at 10 08 32 AM](https://user-images.githubusercontent.com/5834976/165319511-ee1bfe84-8b36-459d-9b53-e1e7345ad017.png)
Control Plane status should not be shown for a hypershift cluster